database/postgres/postgres_test.go

```diff
@@ -120,6 +120,10 @@ func TestMigrate(t *testing.T) {
 			t.Fatal(err)
 		}
 		dt.TestMigrate(t, m)
+		err = d.(*Postgres).Drop()
```

Comment:
Let's leave `TestMigrate()` unchanged and add a new test case for dropping enums e.g. a new `TestDropEnum()` method

Reply:
The reason I threw the drop in `TestMigrate()` is because there needed to be information in the database to drop. So if I were going to create an new method it would look like what I changed TestMigrate to and I feel there is no need for duplicating the code but I see your point.